app/Repositories/Summit/DoctrineSummitEventRepository.php

```diff
@@ -523,6 +525,83 @@ protected function getCustomFilterMappings(int $current_member_id, int $current_
+                        Presentation::ReviewStatusInReview,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND
+                            selp IS NOT null AND (
+                            (selp.submission_lock_down_presentation_status_date IS NOT NULL AND selp.submission_lock_down_presentation_status_date > '$now_utc') OR 
+                            (
+                                (selp.submission_begin_date > '$now_utc' OR selp.submission_end_date < '$now_utc') 
+                                AND 
+                                (selp.selection_begin_date <= '$now_utc' AND selp.selection_end_date >= '$now_utc')
+                            )
+                         )"
+                    ),
+                    Presentation::ReviewStatusPublished => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusPublished,
+                        "(p.status = 'Received' OR p.status = 'Accepted') AND e.published = 1 AND
+                            selp IS NOT null AND ( 
+                            (selp.submission_lock_down_presentation_status_date IS NULL OR selp.submission_lock_down_presentation_status_date <= '$now_utc') AND 
+                            (
+                                (selp.submission_begin_date <= '$now_utc' AND selp.submission_end_date >= '$now_utc') 
+                                OR selp.selection_begin_date > '$now_utc' OR selp.selection_end_date < '$now_utc'
+                            )
+                        )"
+                    ),
+                    Presentation::ReviewStatusAccepted => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusAccepted,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND 
+                        (selp.selection_begin_date > \'%1$s\' OR selp.selection_end_date < \'%1$s\') AND
+                        EXISTS (
+                            SELECT ___sp32.id 
+                            FROM models\summit\SummitSelectedPresentation ___sp32
+                            JOIN ___sp32.presentation ___p32
+                            JOIN ___sp32.list ___spl32 WITH ___spl32.list_type = \'%3$s\' AND ___spl32.list_class = \'%4$s\'
+                            WHERE ___p32.id = e.id 
+                            AND ___sp32.collection = \'%2$s\'
+                        )',
+                        $now_utc,
+                        SummitSelectedPresentation::CollectionSelected,
+                        SummitSelectedPresentationList::Group,
+                        SummitSelectedPresentationList::Session)
+                    ),
+                    Presentation::ReviewStatusRejected => new DoctrineCaseFilterMapping(
+                        Presentation::ReviewStatusRejected,
+                        sprintf('(p.status = \'Received\' OR p.status = \'Accepted\') AND e.published = 0 AND
```

Comment:
this e.published = 0 is not correct check https://github.com/OpenStackweb/summit-api/pull/263/files#r1610638021